ci-test.ps1

```diff
@@ -0,0 +1,41 @@
+param(
+  [Parameter(Mandatory, Position = 0)]
+  [string]$os,
+  [Parameter(Mandatory, Position = 1)]
+  [string]$targetFramework,
+  [Parameter(Mandatory, Position = 2)]
+  [string]$platform,
```

Comment:
nit: this should be named `architecture`

Reply:
It's platform in msbuild.

Reply:
Ok, makes sense here. However, platform is an overloaded term meaning different things in different contexts 😕